miner/miner_test.go

```diff
@@ -75,11 +67,11 @@ func TestCloudBase(t *testing.T) {
 
 	CloudBase(b, bc, w.Public())
```

Comment:
Its still difficult for me to appreciate that this function is mutating the block and adding a new transaction to it. Can we rename it sometime? 🙏